gsa/src/web/pages/usersettings/usersettingspage.js

```diff
@@ -319,6 +328,15 @@ class UserSettings extends React.Component {
     if (isLoading) {
       return <Loading/>;
     };
+    const openVasScanConfigs = scanconfigs.filter(
+        conf => conf.scan_config_type === OPENVAS_SCAN_CONFIG_TYPE);
+    const ospScanConfigs = scanconfigs.filter(
+      conf => conf.scna_config_type === OSP_SCAN_CONFIG_TYPE);
+    const openVasScanners = scanners.filter(
+      scanner => scanner.scanner_type === OPENVAS_SCANNER_TYPE);
+    const ospScanners = scanners.filter(
+      scanner => scanner.scanner_type === OSP_SCANNER_TYPE);
```

Comment:
As above with the scan configs. We should add some filter functions to models/scanner.js